RELEASE_NOTES.md

```diff
@@ -0,0 +1,75 @@
+
+For more information about interchain accounts, check out the [interchain accounts documentation](https://ibc.cosmos.network/main/apps/interchain-accounts/overview).
+
+### Relayer Incentivization
+
+The `fee` module is a self-contained [middleware](https://ibc.cosmos.network/main/ibc/middleware/develop.html) module that extends the base IBC application module. The fee module was designed as an incentivization mechanism to help cover the operational costs of running a relayer (i.e. running full nodes to query transaction proofs and paying for transaction fees associated with IBC packets).
+
+There are three fees within the fee model, one for receiving the packet, one for acknowledging the packet, and one for timeouts. The fees are held in escrow until the packet is either successful or times out. In the case of a successful packet, the timeout fee will be reimbursed, and in the case of an unsuccessful packet, the receiving and acknowledging fee will be reimbursed.
+
+The first version of the fee module only supports incentivization of new channels and existing channels will need to wait for additional functionality to support upgradeability. Using the fee middleware with IBC transactions is optional and acts more like a "tip". End users can manually incentivize IBC packets using the CLI and client developers can leverage the gRPC endpoints to integrate relayer fees within their application.
+
+For more information about fee middleware, check out the [fee middleware documentation](https://ibc.cosmos.network/main/middleware/ics29-fee/overview).
+
+## Additional Changes
+
+### SDK Module Manager
+
+Regen Ledger has historically been using a custom module manager within the application wiring. Regen Ledger `v5.0` migrates from the custom module manager to the latest version of the Cosmos SDK module manager and updates the `ecocredit` module and `data` module accordingly.
+
+### gRPC Error Codes
+
+A community member reported that the gRPC error codes for queries were not being reported correctly, in some cases the error code was `Unknown` and in other cases the error code did not match the standard gRPC error codes. Not all projects building on Regen Ledger will use the error messages provided by Regen Ledger and may choose to create their own error messages based on the error codes returned. To ensure we are providing a good developer experience, we have fixed and updated gRPC error codes to return the expected gRPC error codes.
+
+### Experimental Build
+
+Following Regen Ledger `v4.0`, and now with Regen Ledger `v5.0`, all experimental features that were being developed within the Regen Ledger codebase have been stablilized and included in the stable application build. The experimental application build option has therefore been removed. We will consider a separate release that includes CosmWasm that will be used to reboot Hambach Testnet if developers are wanting to experiment with the latest features alongside CosmWasm contracts.
```

Comment:
```suggestion
Following Regen Ledger `v4.0`, and now with Regen Ledger `v5.0`, all experimental features that were being developed within the Regen Ledger codebase have been stablilized and included in the stable application build. The experimental application build option has therefore been removed. We will consider a separate release that includes CosmWasm that will be used to reboot Hambach Testnet if developers are wanting to experiment with the latest features alongside CosmWasm contracts, otherwise Hambach Testnet will continue running with support for CosmWasm contracts on the experimental build of Regen Ledger v4.0.
```